python/src/scipp/plot/toolbar.py

```diff
@@ -0,0 +1,145 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+# @author Neil Vaytet
+
+import ipywidgets as ipw
+
+
+class PlotToolbar:
+    """
+    Custom toolbar with additional buttons for controlling log scales and
+    normalization, and with back/forward buttons removed.
+    """
+    def __init__(self, canvas=None, ndim=1):
+        # Prepare containers
+        self.container = ipw.VBox()
+        self.members = {}
+
+        # Construct  toolbar
+        if canvas is not None:
+            old_tools = canvas.toolbar.toolitems
+            new_tools = [
+                old_tools[0], old_tools[3], old_tools[4], old_tools[5]
+            ]
+            canvas.toolbar.toolitems = new_tools
+            self.members["mpl_toolbar"] = canvas.toolbar
+            canvas.toolbar_visible = False
+        else:
+            self.add_button(name="menu", icon="bars", tooltip="Menu")
+            self.add_button(name="home",
+                            icon="home",
+                            tooltip="Reset original view")
+        self.add_button(name="rescale_to_data",
+                        icon="arrows-v",
+                        tooltip="Rescale")
+        if ndim > 1:
+            self.add_button(name="swap_axes",
```

Comment:
Would transpose T be better? I mean this in the context only of the two plotting axes - Or is that confusing in a multi-dimensional sense?

Reply:
transpose makes sense in 2d (in fact Simon also suggested to use `.T` as the text inside the button instead of arrows going both ways), but in 3d it cycles through the order of the xyz axes, so it's not exaclty like a transpose.

However, i don't know how much swapping axes in 3d would be used. Also, I then guess "swap" is also not completely accurate, because that would imply a 1 to 1 exchange, when we are performing a cyclic permutation of the axes...
I don't mind changing to `transpose`.

Reply:
I don't see the need for supporting axis swapping in 3d?

Reply:
I guess there isn't really as you can move the camera around.